pip/commands/wheel.py

```diff
@@ -161,87 +156,4 @@ def run(self, options, args):
         if options.build_dir:
             options.build_dir = os.path.abspath(options.build_dir)
 
-        with self._build_session(options) as session:
```

Comment:
I think the session building part should stay on the commands side and the session would be an other argument of the build_wheel function.